scripts/launch_server.sh

```diff
@@ -1,3 +1,3 @@
-mkdir -p photo-data
-docker compose pull
-docker compose up -d
+mkdir -p photo-data build
+docker compose $* pull
+docker compose $* up -d
```

Comment:
We'd want `"$@"` here (with the double quotes) instead of `$*`, so as to handle any future situations where we want to pass in multiple arguments.